scripts/src/modules/focus_element_if_found.js

```diff
@@ -8,7 +8,7 @@
 const focus_element_if_found = (needle_class = "") => {
     const needle_elements = document.getElementsByClassName(needle_class);
 
-    if (!!needle_elements.length) {
+    if (needle_elements.length) {
```

Comment:
The automated linter changed this, but I don't think it should be a problem. Length will either be 0 or a positive value, which will convert to a boolean.

Reply:
It's no issue. We don't need to cast to a boolean - like you said, the value will be evaluated as truthy or falsy anyway.